# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/glibc/loongarch.patch.cross0
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- glibc-2.36/sysdeps/loongarch/preconfigure.vanilla	2022-08-02 12:40:05.190934482 +0200
+++ glibc-2.36/sysdeps/loongarch/preconfigure	2022-08-02 12:40:10.809934258 +0200
@@ -21,7 +21,7 @@
 	abi_flen=64
 	;;
     *)
-	as_fn_error 1 "Unable to determine floating-point ABI!!" "$LINENO" 5
+	abi_flen=64
 	;;
     esac
 
@@ -34,7 +34,8 @@
 	machine=loongarch/lp64
 	;;
     *)
-	as_fn_error 1 "Unable to determine GRLEN!!" "$LINENO" 5
+	grlen=64
+	machine=loongarch/lp64
 	;;
     esac
 
